dg/config/__init__.py

```diff
@@ -203,5 +203,58 @@ def store_credentials(
         with open(dg_credentials_file_path, "w") as credentials_file:
             json.dump(credentials, credentials_file, indent="\t", sort_keys=True)
 
+    def get_dg_settings_file_path(self) -> pathlib.Path:
+        """Returns the path of the settings file
+
+        Returns
+        -------
+        str
+            path of the settings file
+        """
+
+        return pathlib.Path.home() / ".dg" / "settings.json"
+
+    def fyre_commands_hidden(self) -> bool:
```

Comment:
In the server code, we always prefixed boolean functions with verbs (e.g., `is_initialized()`). In this case, I would use `are_fyre_commands_hidden()` or `shall_fyre_commands_be_hidden()`. This is just a suggestion. However, we should introduce a naming convention for boolean functions.